include/dm.h

```diff
@@ -17,79 +23,78 @@
 extern "C" {
 #endif
 
-#include <zephyr/kernel.h>
-#include <stdlib.h>
-#include <zephyr/types.h>
-#include <zephyr/bluetooth/addr.h>
-
 /** @brief Role definition. */
 enum dm_dev_role {
-	/* Undefined role*/
+	/** Undefined role. */
 	DM_ROLE_NONE,
 
-	/* Act as a initiator */
+	/** Act as a initiator. */
 	DM_ROLE_INITIATOR,
 
-	/* Act as a reflector */
+	/** Act as a reflector. */
 	DM_ROLE_REFLECTOR,
 };
 
 /** @brief Ranging mode definition. */
 enum dm_ranging_mode {
-	/* Round trip timing */
+	/** Round trip timing. */
 	DM_RANGING_MODE_RTT,
 
-	/* Multi-carrier phase difference */
+	/** Multi-carrier phase difference. */
 	DM_RANGING_MODE_MCPD,
 };
 
 /** @brief Measurement quality definition. */
 enum dm_quality {
-	/* Good measurement quality */
+	/** Good measurement quality. */
 	DM_QUALITY_OK,
 
-	/* Poor measurement quality */
+	/** Poor measurement quality. */
 	DM_QUALITY_POOR,
 
-	/* Measurement not for use */
+	/** Measurement not for use. */
```

Comment:
```suggestion
	/** Measurement not in use. */
```
Not so sure about this, but maybe it tries to say the measurement is not available?

Reply:
The idea is that the measurement is available (the callback function was called), but the measurement is poor quality, so it should not be used.

Reply:
Then it should say so. For example, Bad measurement quality.

Reply:
Maybe: "Measurements not usable". Because we have a `DM_QUALITY_POOR` field in the enum structure and to avoid confusion.